src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -178,7 +180,14 @@ const ChartValuesView = ({
     const isUpdate = isExternalApp || (commonState.installedConfig?.environmentId && commonState.installedConfig.teamId)
     const validationRules = new ValidationRules()
     const [showUpdateAppModal, setShowUpdateAppModal] = useState(false)
-    
+
+    const isCreateValueFormDirty = isCreateValueView && !!chartValueId && chartValueId === '0' && (!!valueName || commonState.selectedVersion !== selectedVersionFromParent || commonState.chartValues.id !== chartValuesFromParent.id) 
```

Comment:
This condition is not readable try to break it into smaller conditions with meaningful name.